core_backend/app/schemas.py

```diff
@@ -21,6 +21,6 @@ class QuerySearchResult(BaseModel):
     retrieved_title: str
     retrieved_text: str
     retrieved_content_id: int
-    score: float
+    distance: float
```

Comment:
Fixed the naming which had been confusing everyone!

Reply:
ty!